src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -46,11 +62,8 @@ public void configureIndexWithInvalidIndexName() {
 
     @Test
     public void configureIndexExceedingQuota() {
-        ConfigureIndexRequestSpecPod pod = new ConfigureIndexRequestSpecPod().replicas(400);
-        ConfigureIndexRequestSpec spec = new ConfigureIndexRequestSpec().pod(pod);
-        ConfigureIndexRequest configureIndexRequest = new ConfigureIndexRequest().spec(spec);
         try {
-            controlPlaneClient.configureIndex(indexName, configureIndexRequest);
+            controlPlaneClient.configureIndex(indexName, 400);
```

Comment:
I love how much cleaner this is with the new argument patterns!

Reply:
🎉 !